src/Problem/Serializer/ErrorNormalizer.php

```diff
@@ -45,12 +47,24 @@ public function __construct(private readonly bool $debug = false, array $default
      */
     public function normalize(mixed $object, string $format = null, array $context = []): array
     {
+        $identifiers = [
+            'uri_variables' => [
+                'status' => $object instanceof ProblemError ? $object->getStatus() : $object->getStatusCode(),
+            ],
+        ];
+
         $data = [
-            'type' => $context[self::TYPE] ?? $this->defaultContext[self::TYPE],
-            'title' => $context[self::TITLE] ?? $this->defaultContext[self::TITLE],
-            'detail' => $this->getErrorMessage($object, $context, $this->debug),
+            'type' => $object instanceof ProblemError ? $this->iriConverter->getIriFromResource($object, context: $identifiers) : $context[self::TYPE] ?? $this->defaultContext[self::TYPE],
+            'title' => $object instanceof ProblemError ? $object->getTitle() : $context[self::TITLE] ?? $this->defaultContext[self::TITLE],
+            'detail' => $object instanceof ProblemError ? $object->getDetail() : $this->getErrorMessage($object, $context, $this->debug),
+            'instance' => $object instanceof ProblemError ? $object->getInstance() : null,
+            'status' => $object instanceof ProblemError ? $object->getStatus() : $context['statusCode'] ?? $object->getStatusCode(),
```

Comment:
Isn't adding the `status` a BC?

Reply:
Yeah I think adding 'status' and 'instance' breaks the current return responses, I should try to add more checks about the existence of the `extraProperties: ['hydra_errors' => true])` for the current request